client/post-editor/editor-ground-control/index.jsx

```diff
@@ -250,34 +192,14 @@ const EditorGroundControl = React.createClass( {
 			! this.props.isSaveBlocked;
 	},
 
-	isPrimaryButtonEnabled: function() {
-		return ! this.props.isPublishing &&
-			! this.props.isSaveBlocked &&
-			this.props.hasContent
+	canSchedulePost: function() {
```

Comment:
The name of this method doesn't actually match what the code is doing and could lead to confusions later on.